src/App.jsx

```diff
@@ -14,45 +15,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_BY_DEFAULT = 'default';
+const SORT_FIELD_BY_ALPHABET = 'alphabet';
+const SORT_FIELD_BY_LENGTH = 'length';
+
+function getPrepearedGoods(goods, { sortField, reverseStatus }) {
+  const prepearedGoods = [...goods];
+
+  switch (sortField) {
+    case SORT_FIELD_BY_ALPHABET:
+      prepearedGoods.sort((good1, good2) => good1.localeCompare(good2));
+
+      break;
+
+    case SORT_FIELD_BY_LENGTH:
+      prepearedGoods.sort((good1, good2) => (good1.length - good2.length));
+
+      break;
+
+    default:
+      break;
+  }
+
+  if (reverseStatus) {
+    prepearedGoods.reverse();
+  }
+
+  return prepearedGoods;
+}
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortField, setSortField] = useState(SORT_FIELD_BY_DEFAULT);
+  const [reverseStatus, setReverseStatus] = useState(false);
+
+  const visibleGoods
+    = getPrepearedGoods(goodsFromServer, { sortField, reverseStatus });
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={`button is-info ${sortField !== SORT_FIELD_BY_ALPHABET && 'is-light'}`}
+          onClick={() => setSortField(SORT_FIELD_BY_ALPHABET)}
+        >
+          Sort alphabetically
+        </button>
+
+        <button
+          type="button"
+          className={`button is-success ${sortField !== SORT_FIELD_BY_LENGTH && 'is-light'}`}
```

Comment:
better to use classnames package for such cases
<img width="475" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/70220667/92589bde-5bfd-4385-8ba8-d69ae0d6dc20">